hw05-annotations/src/main/java/ru/otus/hw05/TestResult.java

```diff
@@ -0,0 +1,45 @@
+package ru.otus.hw05;
+
+class TestResult {
+
+    private int failedTest;
+
+    private int succeedTest;
+    private int countTest;
+
+    private StringBuilder exceptionDesc = new StringBuilder().append("\n");
+
+    int getFailedTest() {
+        return failedTest;
+    }
+
+    void setFailedTest(int failedTest) {
+        this.failedTest = this.failedTest + failedTest;
+    }
+
+    int getCountTest() {
+        return countTest;
+    }
+
+    void setCountTest(int countTest) {
+        this.countTest = this.countTest + countTest;
+    }
+
+    StringBuilder getExceptionDescription() {
+        return exceptionDesc;
+    }
+
+    void setExceptionDescription(String exceptionDescription) {
```

Comment:
Лучше переименовать в `add*`
```suggestion
    void addExceptionDescription(String exceptionDescription) {
```

А еще лучше явно передать имя метода и описание ошибки. Склеить уже внутри.
```suggestion
    void addExceptionDescription(String methodName, String exceptionDescription) {
```
Так не забудете передать имя метода.